src/mf/mf-uvc.cpp

```diff
@@ -970,10 +972,13 @@ namespace librealsense
                 throw std::runtime_error("Device must be powered to query supported profiles!");
 
             std::vector<stream_profile> results;
-            foreach_profile([&results](const mf_profile& mfp, CComPtr<IMFMediaType> media_type, bool& quit)
-            {
-                results.push_back(mfp.profile);
-            });
+            foreach_profile(
```

Comment:
Is this change still needed or can it be reverted?

Reply:
You're right that it's not needed but I think it's convenient to keep the debug line there, commented.